tools/ClassGen/NodeBuilder.h

```diff
@@ -30,6 +30,73 @@
 #include <vector>
 
 class Builder;
+class NodeBuilder;
+
+class EnumBuilder {
+  /// The name of the enum.
+  std::string name_;
+  /// The fully qualified name of the enum (i.e. with all enclosing namespaces).
+  std::string fullyQualifiedName_;
+  /// The containing namespaces for the enum.
+  std::vector<std::string> namespaces_;
+  /// The enum case names, values, and their corresponding docstrings.
+  std::vector<std::tuple<std::string, int, std::string>> enumCases_;
```

Comment:
I'd kinda prefer a struct with {name, value, docstring} over std::tuple.

Reply:
Fixed in latest push.